pyccl/power.py

```diff
@@ -46,6 +46,19 @@ def sigmaR(cosmo, R):
     return _vectorize_fn(lib.sigmaR, 
                          lib.sigmaR_vec, cosmo, R)
 
+def sigmaV(cosmo, R):
```

Comment:
This is what I meant above, you could just add an optional argument a=1 here and pass it to C.

Reply:
Okay, how exactly do I do this? As in, how do I make the argument optional in python? Or is it better just to change the sigmaR function that is exposed and make it need the use to specify an 'a'?

Reply:
DOne